### PR TITLE
feat: add server url in sentry msg

### PR DESCRIPTION
**Pull request by enguerran**
- 09f069f3 (Enguerran, 38 seconds ago)
    feat: add serverUrl in sentry msg
- 7fb3a925 (Enguerran, 9 minutes ago)
    fix: use sentry dev project
- 7df01552 (Enguerran, 64 minutes ago)
    refactor: rename raven functions

### CODE REVIEW

**Comment by enguerran** [comment]
@y-lohse 
I did not change the way it uses a `let` (mutable) variable to persist the wanted value from gobal state.

But I think my changes around the feature make the code cleaner and clearer. What do you think?

File: src/drive/mobile/lib/reporter.js [hunk 1]
```diff
@@ -2,16 +2,20 @@
 import Raven from 'raven-js'
 
 let isEnable = false
-export const ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/6`
+export const PROD_ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/6`
+export const DEV_ANALYTICS_URL = `https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`
```
Comment (y-lohse) [inline src/drive/mobile/lib/reporter.js]: I don't think we need to export these two.

File: src/drive/mobile/lib/reporter.js [hunk 2]
```diff
@@ -2,16 +2,20 @@
 import Raven from 'raven-js'
 
 let isEnable = false
```
Comment (y-lohse) [inline src/drive/mobile/lib/reporter.js]: Can we rename`isEnable` to `isEnabled` while we're at it please ? :D

File: src/drive/mobile/lib/reporter.js [hunk 3]
```diff
@@ -23,21 +27,23 @@ export const logException = err => {
...
+    Raven.setUserContext = {
+      url: serverUrl
+    }
+    Raven.captureMessage(`[${serverUrl}] ${message}`, {
       level
     })
-    if (updateConfig) {
-      configure(false)
+    if (force) {
+      configureReporter(isEnable)
```
Comment (y-lohse) [inline src/drive/mobile/lib/reporter.js]: Wow, good catch.

File: src/drive/mobile/lib/reporter.js (hunk 1 above)
Comment (enguerran) [reply src/drive/mobile/lib/reporter.js]: :+1:

File: src/drive/mobile/lib/reporter.js (hunk 2 above)
Comment (enguerran) [reply src/drive/mobile/lib/reporter.js]: 👍

**Comment by enguerran** [comment]
[(0,5) [DRI] 📱L'envoi des logs de l'app mobile se fait par utilisateur et pas dans une seule issue Sentry](https://trello.com/c/QU8uqpsn/531-05-dri-%F0%9F%93%B1lenvoi-des-logs-de-lapp-mobile-se-fait-par-utilisateur-et-pas-dans-une-seule-issue-sentry)

**Pull request merged**